crypto/fipsmodule/ec/p256_shared.h

```diff
@@ -50,7 +50,14 @@ typedef unsigned char P256_SCALAR_BYTES[33];
 
 static inline void p256_scalar_bytes_from_limbs(
     P256_SCALAR_BYTES bytes_out, const BN_ULONG limbs[P256_LIMBS]) {
-  OPENSSL_memcpy(bytes_out, limbs, 32);
```

Comment:
I need to go through all the remaining uses of OPENSSL_memcpy and make sure they're OK.